web/templates/setting/form.html.eex

```diff
@@ -0,0 +1,28 @@
+<%= form_for @conn, @action, [as: :settings, method: :put], fn f -> %>
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Value</th>
+      </tr>
+    </thead>
+    <tbody>
+      <%= for svc <- @system_variable_changesets do %>
+        <tr>
+          <td> <%= svc.data.key %> </td>
+
+          <%= inputs_for f, integer_to_atom(svc.data.id), fn ff -> %>
```

Comment:
No you cannot do that cause the user can just have edit and update functionality. then in the controller I read the existing settings and use the ID to organize the changes of the settings. So I don't see how the user can create settings at all